app/controllers/idv/personal_key_controller.rb

```diff
@@ -133,5 +133,13 @@ def redirect_to_retrieve_pii
       user_session[:stored_location] = request.original_fullpath
       redirect_to fix_broken_personal_key_url
     end
+
+    public def step_indicator_step
```

Comment:
Nit: is public necessary here?

```suggestion
    def step_indicator_step
```